docs/contribute/doc.md

```diff
@@ -159,9 +159,8 @@ that declaration will become the body of the tactic doc entry. If there are
 multiple declarations, you can select the one to be used by passing a name to
 the `inherit_description_from` field.
 
-If you prefer a tactic to have a doc string that is different then the doc entry, then between
-the `/--` `-/` markers, write the desired doc string first, then `---` surrounded by new lines,
-and then the doc entry.
+If you prefer a tactic to have a doc string that is different than the doc entry,
+you should write the doc entry as a doc string for the `add_tactic_doc` invocation.
```

Comment:
This change should be made in the docstring of `add_tactic_doc_command` as well. 

Otherwise, looks good to me. Though we'll want to circle back around to the docstrings which do have this `---` separator at some point.

Reply:
> Though we'll want to circle back around to the docstrings which do have this `---` separator at some point.

#2372 :)

Reply:
But good catch, somehow I thought that's what I was reading.